ui/app/AppMain.qml

```diff
@@ -498,45 +495,11 @@ Item {
                 Layout.fillHeight: true
             }
 
-            WalletV2Layout {
-                id: walletV2LayoutContainer
+            UIComponents {
                 Layout.fillWidth: true
                 Layout.alignment: Qt.AlignLeft | Qt.AlignTop
                 Layout.fillHeight: true
             }
-
-            Repeater{
```

Comment:
@alexandraB99 why is this deleted? This must be here and it must be the last children of `StackLayout` component. Without this we won't have `ChatLayout` for each community.